package.json

```diff
@@ -52,6 +54,7 @@
     "react": "16.13.1",
     "react-native": "^0.63.0-rc.1",
     "react-test-renderer": "16.13.1",
+    "react-native-windows": "0.62",
```

Comment:
@kaiguo Should we write "^0.62.0-0"? (to support higher versions later)

Reply:
Yeah `^0.62.0-0` is probably better, it will use the latest 62.